packages/component-library/draft/Kaizen/SplitButton/Dropdown.tsx

```diff
@@ -24,6 +26,7 @@ export default class Dropdown extends React.Component<Props, State> {
   static displayName = "Dropdown"
   static defaultProps = {
     dir: "ltr",
+    primary: false,
```

Comment:
I'm wondering if we should change this to a string? So something like `color: "default" | "primary"`. That way, we get more flexibility if we want to have a third theme.

Reply:
I really love this idea because a primary button and secondary button are exclusive, yet currently can accept both within the component `<Button primary={true} secondary={true}/>`. The intention is not clear. 

I do see this a Kaizen-wide change though. Button and Link would also need to change, and it would be breaking.

Reply:
Thanks for the the comments! Noticed that the `<Tag />` component has a `variant` prop that is more like @jeremy8883 is saying. I took the approach of the `<Button /`> that has the `primary` and `secondary` prop, so the third theme will be the default i guess.

Reply:
Cool cool! I suppose my vote would be to use a `variant` prop, despite the inconsistency with `Button`, but up to you guys.

Reply:
Good call out about Tag, I didn't realise there was already component API discrepancy. 🤦‍♀ In that case, my preference is to move this to a `variant` prop and the DST can work out a plan/pitch for tidying up the API for other components.

Reply:
Remove the `primary` prop and include the `variant` prop. 2 vs 1 🤓 .  Thanks for the comments